### PR TITLE
Add dep5 to bulk license all files

### PR DESCRIPTION
**Pull request by deekshas8**
## What Has Changed?

Created the file .reuse/dep5 in the root of the project to bulk-license the directory.

## Manual Checks?

- [x] Text adheres to the style guide (`vale docs/`)
  - Every sentence is on its own line
  - Headings use [title capitalization](https://capitalizemytitle.com/style/AP/#) (applies also to the sidebar and title of a document)
  - You followed [naming center](https://www.sapbrandtools.com/naming-center/#/dashboard) guidelines when referring to SAP products (e.g. SAP S/4HANA)
- [ ] You checked your spelling and grammar (consider using Grammarly, see `CONTRIBUTING.md`)
- [x] You formatted all changed files with prettier (`npm run prettier`)
- [x] You tested if the documentation still builds (`npm run build`)
- [ ] You verified all new and changed links still work (changing the `id` or name of a file can break links)
- [ ] You have updated the [feature matrix](https://github.com/SAP/cloud-sdk/blob/main/docs/components/data/features.js) if you add documentation on a new feature or otherwise required.

### CODE REVIEW

File: LICENSES/Apache-2.0.txt [hunk 1]
```diff
@@ -187,7 +187,7 @@
       same "printed page" as the copyright notice for easier
       identification within third-party archives.
 
-   Copyright [yyyy] [name of copyright owner]
+   Copyright 2021 SAP SE or an SAP affiliate company and cloud-sdk contributors.
```
Comment (jjtang1985) [inline LICENSES/Apache-2.0.txt:190]:
```suggestion
   Copyright 2018-2021 SAP SE or an SAP affiliate company and cloud-sdk contributors.
```

File: .reuse/dep5 [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+ project’s code for that SAP External Product, subject to the terms of such
+ license agreement. If you do not have a valid license agreement for the use of
+ a particular SAP External Product, then you may only make use of any API Calls
+ in this project for that SAP External Product for your internal, non-productive
+ and non-commercial test and evaluation of such API Calls. Nothing herein grants
+ you any rights to use or access any SAP External Product, or provide any third
+ parties the right to use of access any SAP External Product, through API Calls.
+
+Files: *
+Copyright:  2021 SAP SE or an SAP affiliate company and cloud-sdk contributors.
```
Comment (jjtang1985) [inline .reuse/dep5:28]:
```suggestion
Copyright:  2018-2021 SAP SE or an SAP affiliate company and cloud-sdk contributors.
```

**Review by jjtang1985 — CHANGES_REQUESTED** [review]
Looks good 👍 
I provided some minor suggestions as we started this repo earlier.
Let's see whether the nightly job will be happy with it.

**Comment by jjtang1985** [comment]
@deekshas8 , please close this [issue](https://github.com/SAP/cloud-sdk/issues/397) once the PR is merged so we can wait for the nightly job.

File: LICENSES/Apache-2.0.txt (hunk 1 above)
Comment (deekshas8) [reply LICENSES/Apache-2.0.txt:190]: The copyright text in https://github.com/SAP/cloud-sdk/blob/main/LICENSE file has the following text as well 
"Copyright 2021 SAP SE or an SAP affiliate company and cloud-sdk contributors." 
Do I need to update it here as well?

Comment (jjtang1985) [reply LICENSES/Apache-2.0.txt:190]: Good point. Please go ahead and update it as well.

**Review by jjtang1985 — APPROVED** [review]

**Pull request merged**